browser/brave_content_browser_client.cc

```diff
@@ -220,6 +220,7 @@ using extensions::ChromeContentBrowserClientExtensionsPart;
 #if !BUILDFLAG(IS_ANDROID)
 #include "brave/browser/new_tab/new_tab_shows_navigation_throttle.h"
 #include "brave/browser/ui/geolocation/brave_geolocation_permission_tab_helper.h"
+#include "brave/browser/ui/webui/brave_education/brave_education_ui.h"
```

Comment:
same as above, do not use os buildflags to guard features that are disabled on some platforms.